akka-sample-cluster-client-grpc-java/README.md

```diff
@@ -1,7 +1,7 @@
 # Cluster Client with gRPC transport
 	
 See purpose of this example and important clarifications of when to use this approach in
-[Migration to Akka gRPC](https://doc.akka.io/docs/akka/2.6/cluster-client.html#migration-to-akka-grpc).
+[Migration to Akka gRPC](https://pekko.apache.org/docs/pekko/current/cluster-client.html#migration-to-akka-grpc).
```

Comment:
should be https://pekko.apache.org/docs/pekko/current/cluster-client.html#migration-to-apache-pekko-grpc

Reply:
I'll let it slide but it isn't good practice to update every so many files in a PR, could you restrict future PRs to concentrate on the dirs relating the actual samples you are changing?

Reply:
updated